manager/allocator/network.go

```diff
@@ -165,6 +135,10 @@ func (a *Allocator) doNetworkInit(ctx context.Context) (err error) {
 		return err
 	}
 
+	if err := a.allocateNetworks(ctx, false); err != nil {
```

Comment:
Please add a comment on why we do it in this order i.e. existingOnly=true first and existingOnly=false later.

Reply:
The comments are added in detail in allocateNetworks and the usage of existingOnly flags